packages/twenty-front/src/modules/ui/field/display/components/BooleanDisplay.tsx

```diff
@@ -15,7 +13,7 @@ type BooleanDisplayProps = {
 };
 
 export const BooleanDisplay = ({ value }: BooleanDisplayProps) => {
-  if (!isDefined(value)) {
+  if (value === null || value === undefined) {
```

Comment:
Why ? Is it different than !isDefined ?

Reply:
because of storybook build failing:
BooleanDisplay is in vyw + using twenty-shared, this seems to be an issue currently. We should investigate but I'm protecting the devXP

Reply:
What kind of error had you when building over this ? Emotion.styled is not a function one ?